### ISSUE TITLE
Use xspec.io domain

### ISSUE THREAD

**Issue by rolfkleef** [xspec/xspec.github.io#4]
[Per comment](https://github.com/xspec/xspec/issues/600#issuecomment-528612802) by @cmarchand:

I've bought xspec.io for this, and I'll be happy to give this to XSpec community

[And](https://github.com/xspec/xspec/issues/600#issuecomment-581141594)

Done. I'm able change DNS when you want.

**Comment by cmarchand** [comment]
Give me DNS informations, I'll update them at gandi.net. You can send them by private mail.

**Comment by cirulls** [comment]
The procedure to map xspec.github.io to xspec.io is described [here](https://help.github.com/en/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain). I did up to point 3 which created a [CNAME file](https://github.com/xspec/xspec.github.io/blob/master/CNAME) at the root of the repo. 

@cmarchand: you would need to do point 4 which requires the creation of an `ALIAS` or `ANAME` record or alternatively an `A` record at gandi.net. If you create an `ALIAS` or `ANAME` record you can use `xspec.github.io`, if you use an `A` record, you can find the IP addresses for GitHub Pages on the link.     

Let me know when it's done and I'll pick it up from there.

**Comment by cmarchand** [comment]
Done. Now, we have to wait...
Christophe

**Comment by cmarchand** [comment]
Online.
http://xspec.io/
I think we can close this.

**Comment by cirulls** [comment]
Brilliant! Thanks @cmarchand, I confirm the URL works at my end too. 

I would still like to do point 7 and 8 in [this list](https://help.github.com/en/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain) before closing this ticket. Regarding point 8 (enforce HTTPS), I'm currently getting this message when I try to enable it:

![image](https://user-images.githubusercontent.com/13256236/86219912-c7c1ea00-bb7a-11ea-80d9-491c56f98e55.png)

I'll wait 24 hours before checking again. I also need to check that we're not serving mixed content (http and https) in any link and change the `baseurl` in [config.toml](https://github.com/xspec/xspec.github.io/blob/hugo/config.toml#L2).

**Comment by cirulls** [comment]
I enforced HTTPS as the option is now available: https://xspec.io

There is mixed content as shown by the warning next to the padlock so I'm going to review all the links in the website.

![image](https://user-images.githubusercontent.com/13256236/86389738-b407a800-bc8e-11ea-89ef-5dac7b046c47.png)

**Comment by cirulls** [comment]
I completed all the points in the instructions and merged #6. Mixed content is also fixed and the warning on the padlock has disappeared. I also set up the `hugo` branch as the default branch so that PR are automatically raised against this branch and configured the `hugo` and `master` branches as protected so that they cannot be accidentally deleted.

![image](https://user-images.githubusercontent.com/13256236/86567874-ee5d9780-bf63-11ea-97e7-8db39062e7d0.png)

I'm closing this issue, thanks @cmarchand and @rolfkleef for your work!
